index.html

```diff
@@ -279,9 +279,15 @@ <h2>Contact me</h2>
               love to help with it.
             </p>
             <div class="tonic-courses">
-              <input type="text" placeholder="Mack donald" />
-              <input type="text" placeholder="mackdo@gmail.com" />
-              <textarea name="contact" id="msg" cols="30" rows="10">
+              <input type="text" placeholder="Mack donald" required />
```

Comment:
- [ ] I kindly suggest you to add  `maxlength="30"` attribute to name input so that users could not input a couple of random letters 😄